Lombiq.Privacy.csproj

```diff
@@ -1,9 +1,12 @@
 <Project Sdk="Microsoft.NET.Sdk.Razor">
 
+  <Import Condition="'$(NuGetBuild)' != 'true'" Project="..\..\Utilities\Lombiq.NodeJs.Extensions\Lombiq.NodeJs.Extensions\Lombiq.NodeJs.Extensions.props" />
+
   <PropertyGroup>
     <TargetFramework>net6.0</TargetFramework>
     <AddRazorSupportForMvc>true</AddRazorSupportForMvc>
     <DefaultItemExcludes>$(DefaultItemExcludes);.git*;node_modules\**;Tests\**</DefaultItemExcludes>
+    <NodeJsExtensionsCreateStylelintConfigurationFile Condition="'$(NuGetBuild)' != 'true'">false</NodeJsExtensionsCreateStylelintConfigurationFile>

```

Comment:
Hm, currently a full assets build consists of both linting and compiling. Are you saying that we need to introduce an option to disable the linter during a build?

Reply:
Indeed, and yes, we need such an option. Previously, this wasn't an issue since on we could just disable linting from stylelintrc globally if we only wanted SCSS build; per-project build wasn't a concern.

Reply:
Alright then.